static/js/app.js

```diff
@@ -109,6 +109,8 @@ function shareByEmail() {
 
 // check newsletter form submission
 const newsletterForm = document.getElementById('newsletterForm');
+
+if(newsletterForm != null) {
```

Comment:
Looks great. what do you think if you do the same as i did for the form https://github.com/rotationalio/rotational.io/blob/ca396dc398edbc617a0b544eaf59d8b9edce4474/static/js/app.js#L12
Being consistent would be great. Or use the same conditional check as you need here by removing the optional chaining on form element

Reply:
I made the update to use optional chaining for consistency. Thanks!